etl/steps/data/garden/regions/2023-01-01/regions.yml

```diff
@@ -0,0 +1,1054 @@
+# Region definitions (see more details in the description of the 'regions' garden step).
+# Each region must contain the following fields:
+# code: Region code (unique for each region).
+# name: Region name.
+#
+# Additionally, each region can contain the following fields:
```

Comment:
(We could have JSON schema validation to make sure we're not making mistakes when manually editing the file, but that could be added in the future)